encoding/rs/frame.go

```diff
@@ -33,3 +37,61 @@ func Decode(b []byte) (Frame, error) {
 	}
 	return f, nil
 }
+
+// EncodeFrames serializes a slice of frames into a byte slice.
+func EncodeFrames(frames []*Frame) ([]byte, error) {
```

Comment:
Why don't we use existing serialization methods like [this](https://github.com/Layr-Labs/eigenda/blob/master/core/data.go#L96)?

Reply:
From split encoder perspective we want to serialize the proofs and the coefficients separately. From node perspective we could make sure it receives the chunks in the expected serialized format.

Reply:
@dmanc requested that the chunk store had the capability of uploading the `encoding.Proof` objects and `rs.Frame` objects separately. The code you point to is capable of serializing both at the same time, but does provide a way to serialize/deserialize them separately.

Reply:
I see. How about [this one](https://github.com/Layr-Labs/eigenda/blob/master/core/data.go#L376)?